### ISSUE TITLE
Wrong title in og:title and twitter:title

### ISSUE THREAD

**Issue by fwhamm** [steamlead/grav-plugin-autoseo#3]
I recently installed Auto SEO but I recognized that on all pages og:title and twitter:title are not correctly filled. Instead of the page's title they are filled with something that looks like a slug (but isn't).

My [installation](http://blog.frankhamm.work/):
* Auto SEO version: 1.3.2
* Grav version: 1.3.10
* Asustor NAS with PHP Version 5.6.30

You can see it on the Game Coder site, e.g. [Revue de cours UE4](https://www.gamecoderblog.com/fr/unreal-engine4/courses-review). Here are some lines from the HTML source code:

```
<title>Revue de cours UE4 | Game Coder Blog</title>
<meta name="og:sitename" property="og:sitename" content="Game Coder Blog" />
<meta name="og:title" property="og:title" content="revue-de-cours-ue4" />
<meta name="twitter:title" property="twitter:title" content="revue-de-cours-ue4" />
```

I'm sorry if I unintentionally annoyed, this is my first post on Github

**Comment by LaurentOngaro** [comment]
Hi, I'll look at that as soon as possible.

**Comment by LaurentOngaro** [comment]
Hi, I've fixed this point in the release 1.3.3 I've just published.
Thanks for your report and your patience.

**Comment by fwhamm** [comment]
Thanks for the fix!
